rep-2001.rst

```diff
@@ -763,14 +764,24 @@ Simulation
       extends:  [ros_base]
       packages: [ros_ign_bridge, ros_ign_gazebo, ros_ign_image, ros_ign_interfaces]
 
+Dev Tools
+"""""""""
+
+::
+
+  - dev_tools:
+      packages: [python3-colcon-common-extensions, python3-flake8, python3-pip,
```

Comment:
I'm curious about the rationale for including flake8 (and to a lesser degree pytest and pytest-cov) when these aren't ubiquitous dependencies for a ROS developer workflow.

Reply:
In short, they're on the list here: https://docs.ros.org/en/humble/Installation/Alternatives/Ubuntu-Development-Setup.html#install-development-tools-and-ros-tools

Personally, I would just go with `build-essential cmake git python3-colcon-common-extensions python3-pip python3-rosdep python3-setuptools python3-vcstool` but I was deferring to whoever wrote the installation page.

Reply:
> I'm curious about the rationale for including flake8 (and to a lesser degree pytest and pytest-cov) when these aren't ubiquitous dependencies for a ROS developer workflow.

I'd honestly go even further than this, and add in the packages corresponding to the flake8 plugins that we expect to be there.  We've had lots and lots of problems where people have trouble getting the right combination of flake8 plugins to match CI, and get frustrated by this.  Adding them here won't totally solve it (if people are using pip installed packages, or venvs they could still have the problem), but it will at least help.